suite-native/app/app.config.ts

```diff
@@ -67,16 +67,6 @@ const projectIds = {
 } as const satisfies Record<BuildType, string>;
 
 const buildType = (process.env.EXPO_PUBLIC_ENVIRONMENT as BuildType) ?? 'debug';
-const isCI = process.env.CI == 'true' || process.env.CI == '1';
-
```

Comment:
what's wrong with this? it already saved out ass once or twice I think :D

Reply:
I've tried to explain it in commit message - Detox build was broken because expo-updates is evaluating app.config.ts even though it's not necessary for Dextox build. I belive that if those ENV variables are missing, we will find that soon anyway? https://github.com/trezor/trezor-suite/actions/runs/11483547009/job/31959232440?pr=14530#step:10:3143